### PR TITLE
[big-chonker][frontend] battle pass and other live dashboard components 

### PR DESCRIPTION
**Pull request by abigger87**
Battle pass updates
New Styles
Modified Profile type in ~/odyssey.d.ts to include a qrCodeId field of type string

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -22,6 +22,7 @@
   "dependencies": {
     "@auth0/auth0-spa-js": "^1.2.4",
     "@babel/register": "^7.6.2",
+    "@next/bundle-analyzer": "^9.2.1",
```
Comment (WilhelmWillie) [inline package.json]: what's this?

Comment (abigger87) [reply package.json]: Analyzes package sizes

Comment (abigger87) [reply package.json]: We can remove thought it might be useful

**Comment by WilhelmWillie** [comment]
Can you include some screenshots of what this looks like?

File: components/BattlePass.tsx [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+      </MarginDiv>
+    </TableTD>
+  );
+};
+
+const BattlePass: React.FunctionComponent<Props> = props => {
+  const { profile } = props;
+
+  // * useBattlePass Custom Hook
+  const bp = useBattlePass(profile);
```
Comment (WilhelmWillie) [inline components/BattlePass.tsx]: nice

File: lib/useBattlePass.tsx [hunk 3]
```diff
@@ -0,0 +1,40 @@
+import { useEffect, useState } from "react";
+import { Service } from "../types/Service";
+
+interface BPObject {
+  id?: string;
```
Comment (jzeiders) [inline lib/useBattlePass.tsx]: Is there a reason everything is an optional?

Comment (abigger87) [reply lib/useBattlePass.tsx]: Um kinda because i didnt want to populate my local db so i didnt have all the data yet

**Comment by abigger87** [comment]
Posted Screenshots in slack.
chrome-capture (1)

**Comment by jzeiders** [comment]
Since it's a prototype will merge if it passes E2E but not deepscan.

**Comment by abigger87** [comment]
LGTM

**Pull request merged**